docs/model/introduction-to-model-architecture.md

```diff
@@ -24,7 +24,10 @@ Model is mostly divided into three parts: Entity, Repository and Facade.
 ## Entity
 Entity is a class encapsulating data. All entities are persisted by Doctrine ORM. One entity class usually represents one table in the database and one instance of the entity represents one row in the table. The entity is composed of fields, which can be mapped to columns in the table. Doctrine ORM annotations are used to define the details about the database mapping (types of columns, relations, etc.).
 
-Entities are inspired by Rich Domain Model. That means entity is the place where domain logic belongs (e.g. `Product::changeVat()` sets vat and marks product for price recalculation).
+Entities are inspired by Rich Domain Model. That means entity is usually the place where domain logic belongs (e.g. `Product::changeVat()` sets vat and marks product for price recalculation).
+However, the rule applies only to the situations when there is no external dependency required. In other words, entities should deal with their own data only and must not be dependent on any other services,
```

Comment:
what about `applies only for the situations where/if there is` ?

Reply:
ok, `where` it is :slightly_smiling_face: